Description: Set Ubuntu PackageKit vendor information
Forwarded: not-needed
Author: Matthias Klumpp <mak@debian.org>
Last-Update: 2013-12-04
--- a/etc/Vendor.conf
+++ b/etc/Vendor.conf
@@ -1,8 +1,6 @@
 # Only the system vendor should modify this file, ordinary users
 # should not have to change anything.
 
-# Distributions should patch or replace this file when generating packages
-
 [PackagesNotFound]
 
 # This is a link to a vendor URL which is shown when a suitable package
@@ -12,7 +10,7 @@
 # If the value is set to 'none' then no link is shown.
 #
 # default=http://www.packagekit.org/pk-package-not-found.html
-DefaultUrl=http://www.packagekit.org/pk-package-not-found.html
+DefaultUrl=https://help.ubuntu.com/community/Repositories/
 
 # The URL which is shown to the user when a codec could not be found.
 # It should explain why certain codecs cannot be used, and perhaps show
@@ -21,7 +19,7 @@
 # If the value is set to 'none' then the value of DefaultUrl is used.
 #
 # default=none
-CodecUrl=none
+CodecUrl=http://shop.canonical.com/index.php?cPath=19&osCsid=f1e370ea7563ed5e654c10450364ff24
 
 # The URL which is shown to the user when hardware drivers could not be found.
 # It should explain why some hardware is not supported, and links to futher
